x/ugov/keeper/keeper.go

```diff
@@ -4,23 +4,29 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/keeper"
+
 	"github.com/umee-network/umee/v6/x/ugov"
 )
 
 var _ ugov.Keeper = Keeper{}
 
 // Builder constructs Keeper by perparing all related dependencies (notably the store).
 type Builder struct {
-	storeKey storetypes.StoreKey
-	Cdc      codec.BinaryCodec
+	storeKey              storetypes.StoreKey
+	Cdc                   codec.BinaryCodec
+	BankKeeper            keeper.BaseKeeper
+	enableLiquidatorQuery bool
 }
 
 func NewBuilder(
-	cdc codec.BinaryCodec, key storetypes.StoreKey,
+	cdc codec.BinaryCodec, key storetypes.StoreKey, bk keeper.BaseKeeper, enableLiquidatorQuery bool,
```

Comment:
Since this query is coming in the next version of SDK, we can enable it by  default  - we don' need `enableLiquidatorQuery`

Reply:
ok